### DISCUSSION TITLE
Отображение Input Source непосредственно у края поля.

### DISCUSSION THREAD

**Discussion by roman-pinchuk** [freefelt/YouType#29] [Ideas]
Есть предложение добавить новую функциональность в виде отображение Источника Ввода возле поля ввода.
На картинке ниже я примерно показал о чём речь:
![hint-input-source](https://user-images.githubusercontent.com/43293123/233430049-1941117f-ba55-4e3c-9331-05ebafb80381.jpg)
Иногда курсор с иконкой источника перекрывает вводимый текст.

**Comment by freefelt** [comment]
Спасибо за предложение! В принципе, подобное должно быть можно реализовать, но только с включенным расширенным режимом
